src/main/java/com/minecolonies/rankup/modules/databases/config/DatabaseConfig.java

```diff
@@ -0,0 +1,31 @@
+package com.minecolonies.rankup.modules.databases.config;
+
+import com.minecolonies.rankup.internal.configurate.BaseConfig;
+import ninja.leaping.configurate.objectmapping.Setting;
+import ninja.leaping.configurate.objectmapping.serialize.ConfigSerializable;
+
+/**
+ * The database section to the main config file.
+ */
+@ConfigSerializable
+public class DatabaseConfig extends BaseConfig
+{
+    @Setting(comment = "You may choose between MySQL (Mariadb works too) or H2 (Hikari) databases for Player Stats storage!")
+    public String database = "h2";
+
+    @Setting
+    public String sqlUsername = "user";
+
+    @SuppressWarnings("squid:S2068")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:S2068' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1309)